webpack.config.dev.js

```diff
@@ -17,7 +17,7 @@ module.exports = webpackMerge.smart(config, {
             patterns: [
               {
                   from: "node_modules/@guardian/commercial/dist/bundle",
-                  to: "commercial/[name].[ext]"
+                  to: "commercial"
```

Comment:
If they are the same could the copy steps move from the dev and prod scripts to `webpack.config.js` ?